jasmine/jasmine.d.ts

```diff
@@ -180,9 +182,12 @@ declare namespace jasmine {
         addMatchers(matchers: CustomMatcherFactories): void;
         specFilter(spec: Spec): boolean;
         throwOnExpectationFailure(value: boolean): void;
-        seed: (s: number) => number;
-        randomTests: () => boolean;
-        randomizeTests: (b: boolean) => void;
+        seed(seed: string | number): string | number;
```

Comment:
is `seed` guaranteed to return `number` if `number` is passed?

if so, it should be split into
```ts
seed(seed: number): number;
seed(seed:  string): string ;
```